### PR TITLE
Part 1/n migration steps for kotlin migration

### PR DESCRIPTION
**Pull request by reidbaker**
Following https://developer.android.com/build/migrate-to-kotlin-dsl
1/n Add parentheses to method calls 

Should be a no-op change. If you see a behavioral change please flag it. 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Review by bartekpacia — APPROVED** [review]
Nice.

I think you could also replace all single-quotes with double-quotes in the same PR (if there are any).

**Comment by johnmccutchan** [comment]
test-exempt: cleanup

**Comment by reidbaker** [comment]
> Nice.
> 
> I think you could also replace all single-quotes with double-quotes in the same PR (if there are any).

That is the next pr, Trying to keep them small.

File: packages/flutter_tools/gradle/src/main/groovy/flutter.groovy [hunk 1]
```diff
@@ -273,14 +273,14 @@ class FlutterPlugin implements Plugin<Project> {
...
+                        multiDexKeepFile(project.file(flutterMultidexKeepfile))
                     }
                 }
             }
             project.dependencies {
-                implementation "androidx.multidex:multidex:2.0.1"
+                implementation("androidx.multidex:multidex:2.0.1")
+            }
             }
-        }
```
Comment (gmackall) [inline packages/flutter_tools/gradle/src/main/groovy/flutter.groovy:283]: looks like the spacing got messed up here

File: packages/flutter_tools/gradle/src/main/groovy/flutter.groovy [hunk 2]
```diff
@@ -544,27 +544,27 @@ class FlutterPlugin implements Plugin<Project> {
...
     private void configurePluginDependencies(Object dependencyObject) {
-        assert dependencyObject.name instanceof String
+        assert(dependencyObject.name instanceof String)
         Project pluginProject = project.rootProject.findProject(":${dependencyObject.name}")
         if (pluginProject == null ||
             !doesSupportAndroidPlatform(pluginProject.projectDir.parentFile.path)) {
             return
-        }
-        assert dependencyObject.dependencies instanceof List
+            }
```
Comment (gmackall) [inline packages/flutter_tools/gradle/src/main/groovy/flutter.groovy:552]: Was this indent intentional?

File: packages/flutter_tools/gradle/src/main/groovy/flutter.groovy [hunk 3]
```diff
@@ -544,27 +544,27 @@ class FlutterPlugin implements Plugin<Project> {
...
+            assert(pluginDependencyName instanceof String)
             if (pluginDependencyName.empty) {
                 return
             }
             Project dependencyProject = project.rootProject.findProject(":$pluginDependencyName")
             if (dependencyProject == null ||
                 !doesSupportAndroidPlatform(dependencyProject.projectDir.parentFile.path)) {
                 return
-            }
+                }
```
Comment (gmackall) [inline packages/flutter_tools/gradle/src/main/groovy/flutter.groovy:563]: Also this indent

**Review by gmackall — APPROVED** [review]
LGTM outside of some spacing questions

File: packages/flutter_tools/gradle/src/main/groovy/flutter.groovy (hunk 2 above)
Comment (christopherfujino) [reply packages/flutter_tools/gradle/src/main/groovy/flutter.groovy:552]: ping

**Review by christopherfujino — APPROVED** [review]
LGTM with nit about the indentation

File: packages/flutter_tools/gradle/src/main/groovy/flutter.groovy (hunk 2 above)
Comment (reidbaker) [reply packages/flutter_tools/gradle/src/main/groovy/flutter.groovy]: fixed

**Pull request merged**